cellarium/cas/client.py

```diff
@@ -201,6 +202,17 @@ def validate_and_sanitize_input_data(
                     f"CAS schema but in a different order. The input features will be reordered according to "
                     f"'{feature_schema_name}'"
                 )
+            if e.incompatible_x_type is not None:
+                self._print(
+                    f"CAS expects the input data matrix to be of type 'float32', but the data provided is of type "
+                    f"'{e.incompatible_x_type}'. If possible, the input data will be converted to 'float32'."
+                )
+            if e.incompatible_total_mrna_umis_type is not None:
```

Comment:
I left a comment about `total_mrna_umis` in `validate` function, this question also applies to here.